docs/ref/settings.txt

```diff
@@ -65,9 +65,10 @@ See :doc:`/howto/error-reporting` for more information.
 Default: ``[]`` (Empty list)
 
 A list of strings representing the host/domain names that this Django site can
-serve. This is a security measure to prevent an attacker from poisoning caches
-and triggering password reset emails with links to malicious hosts by submitting
-requests with a fake HTTP ``Host`` header, which is possible even under many
+serve. This is a security measure to prevent HTTP Host header attacks, in which
```

Comment:
You could add a link to the discussion in `docs/topics/security.txt`:
```
:ref:`HTTP Host header attacks <host-headers-virtual-hosting>`
```
Also, I think a comma isn't needed before "in which".